### ISSUE TITLE
Add NextJS Template

### ISSUE THREAD

**Issue by ixartz** [michaelrambeau/bestofjs#389]
GitHub project: https://github.com/ixartz/Next-js-Blog-Boilerplate
Link to live demo: https://creativedesignsguru.com/demo/Nextjs-Blog-Boilerplate/

Best of JavaScript is for the most popular open-source projects related to the web platform and Node.js.

Before submitting the request to add a new project, please check the following points:

- [X] The project is related to the web platform or Node.js
- [X] The project has more than 100 stars on GitHub
- [X] The project has thorough documentation in English
- [X] The project is not deprecated and actively maintained

How would this project be tagged? (_Pick one or several existing tags_)

React, Static site generator, Boilerplate, Blog

**Comment by michaelrambeau** [comment]
Hello Remi @ixartz 
Thank you for the suggestion.
I checked Next.js site and I'm curious about the difference between this project and the official "TypeScript Blog Starter" that you can find here:

https://github.com/vercel/next.js/tree/canary/examples/blog-starter-typescript

Both use TypeScript and Tailwind CSS but it seems yours has more features "under the hood", especially about SEO. Maybe it could be interesting to explain about these differences?
In other words, it could be interesting to get arguments to pick this boilerplate rather than the official one that is maintained by the Vercel company.

Thank you in advance for your feedback!

**Comment by ixartz** [comment]
Hi Michael,

Thank you for your message.

If I'm not mistaken, here is the major difference.
Next Blog Boilerplate is more oriented to developer with:
- Syntax highlight for code example in markdown file
- ESLint for code linter
- Prettier for code formatter
- Integrate with husky and lint-staged for keeping the code quality high
- Bundle analyzer
- SEO friendly
- Minimalist blog theme
- Deployment to Netlify in on click (optional)


The theme is a little subjective and it's my personal point of view. I prefer a minimalist theme where I can my own thing on top of it, more easy to customize it to meet my needs.
The official theme uses big pictures/photos, I think the theme is more oriented to designer or not really for developer. By using big pictures/photos for each article, you need to find an image which integrate well in your design. It can be hard for developer who don't have a visual/design skill (like myself).

At the end, we are complementary, each templates has his pros/cons.

Let me know if you need more information.

**Comment by michaelrambeau** [comment]
Thank you for the feedback Remi, OK, `Next js Blog Boilerplate` will be available on _Best of JS_ very soon.

**Comment by michaelrambeau** [comment]
Hello Remi @ixartz 
`Next js Blog Boilerplate` is now available on _Best of JS_.
Thank you for your feedback!

**Comment by ixartz** [comment]
Thank you @michaelrambeau
